FreeRTOS/FreeRTOS-Plus-TCP
# Replace strcpy with strncpy

Description
-----------
This PR replaces strcpy with strncpy in the codebase.

Test Steps
-----------


Checklist:
----------


    - Tested with DNS demo on hardware. 

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
